lib/avalon/batch/manifest.rb

```diff
@@ -82,19 +124,28 @@ def load!
 
       def start!
         File.open("#{@file}.processing",'w') { |f| f.puts Time.now.xmlschema }
+        manifest_logger.info("Processing started #{@file}")
```

Comment:
Yeah, I guess so ... (#$&&@!, this increases the scope of work, since almost nothing in Avalon is i18n)